projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/title-optimization/title-optimization-options.scss

```diff
@@ -19,13 +19,21 @@
 		}
 	}
 
-
 	&-content {
 		display: flex;
 		flex-direction: column;
 	}
 
 	&-label {
+		font-size: 15px;
 		font-weight: 600;
+		line-height: 22px;
```

Comment:
Since the text is inside the label now, maybe a `cursor: pointer;` would make it more obvious.